receiver/m365receiver/metadata.yaml

```
@@ -0,0 +1,205 @@
+      aggregation: cumulative
+    attributes: []
+  m365.teams.device_usage.count:
+    enabled: true
+    description: The number of unique users by device/platform that have used Teams in the last 7 days.
+    unit: "{users}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: [teamsDevices]
+  m365.teams.calls.count:
+    enabled: true
+    description: The number of MS Teams calls from users in the organization in the last 7 days.
+    unit: "{calls}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.teams.meetings.count:
+    enabled: true
+    description: The number of MS Teams meetings for users in the organization in the last 7 days.
+    unit: "{meetings}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.teams.message.team.count:
+    enabled: true
+    description: The number of MS Teams team-messages sent by users in the organization in the last 7 days.
+    unit: "{messages}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.teams.messages.private.count:
+    enabled: true
+    description: The number of MS Teams private-messages sent by users in the organization in the last 7 days.
+    unit: "{messages}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.onedrive.files.count:
+    enabled: true
+    description: The number of total files across the OneDrive for the last 7 days.
+    unit: "{files}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.onedrive.files.active.count:
+    enabled: true
+    description: The number of active files across the OneDrive in the last 7 days.
+    unit: "{files}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.onedrive.user_activity.count:
+    enabled: true
+    description: The number of users who have interacted with a OneDrive file, by action, in the last 7 days.
+    unit: "{users}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: [onedriveActivity]
+  m365.outlook.mailboxes.active.count:
+    enabled: true
+    description: The number of mailboxes that have been active each day in the organization for the last 7 days.
+    unit: "{mailboxes}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.outlook.email_activity.count:
+    enabled: true
+    description: The number of email actions by members over the period of time in the organization Outlook.
+    unit: "{emails}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: [outlookActivity]
+  m365.outlook.storage.count:
```

Same here, I think used instead of count makes more sense.
```suggestion
  m365.outlook.storage.used:
```